packages/utils/src/lib/log-results.ts

```diff
@@ -0,0 +1,53 @@
+type PromiseFulfilledResult<T> = {
+  status: 'fulfilled';
+  value: T;
+};
+
```

Comment:
This type is already in the global namespace.

```suggestion
```

![image](https://github.com/code-pushup/cli/assets/34691111/21a270c0-4b11-453c-882c-b64bdddbd793)

Reply:
Removed